ui/app/AppLayouts/Communities/panels/SortableTokenHoldersList.qml

```diff
@@ -181,14 +190,31 @@ StatusListView {
         background: Item {
             Rectangle {
                 anchors.fill: parent
-
                 anchors.topMargin: delegate.topPadding
 
                 radius: Style.current.radius
                 color: (delegate.hovered || delegate.ListView.isCurrentItem)
                        ? Theme.palette.baseColor2 : "transparent"
             }
 
+            Rectangle {
+                anchors.fill: parent
+                anchors.topMargin: delegate.topPadding
+
+                radius: Style.current.radius
+                color: "transparent"
+
+                SequentialAnimation on color {
```

Comment:
There is no spacing between rows

<img width="1562" alt="Screenshot 2023-08-03 at 22 52 31" src="https://github.com/status-im/status-desktop/assets/47811206/f3115c38-83aa-4fc4-ba97-b9b961e0ddfb">

Reply:
Thank you @alexjba for pointing that. Fixed!
And figma links added.

[Kazam_screencast_00002.webm](https://github.com/status-im/status-desktop/assets/20650004/9a2d9efe-cf34-4a18-8061-3567f626d5c1)